libraries/AP_RPM/AP_RPM.cpp

```diff
@@ -113,10 +116,12 @@ void AP_RPM::convert_params(void)
     bool type_set;
     uint8_t rpm_type = 0;
     uint8_t rpm2_type = 0;
+    uint8_t rpm3_type = 0;
```

Comment:
I'm still not sure because it's my first Pull & Request.

Thank you for your answer.

You mean simply adding a variable to RPM is not the solution.

I thought I could modify the functions of the two RPM sensors, but there's no solution yet.

Do you think positively about how to expand RPM sensors based on instances?